### ISSUE TITLE
Project Find Broken in Upgrade

### ISSUE THREAD

**Issue by barrettab** [phistrom/basecampy3#23]
When upgrading form 0.3.0 >> 0.6.0 the find function seems to have broke in the process. 
I used to use 
`bc3.projects.find("Project Name')` 
in existing code which would return a list of matched project names but this seems to have broke with no other changes to the code after updating.  And is returning an error "AttributeError: 'str' object has no attribute 'search' " Let me now if there's any additional info I can provide.  

Sample Code:

```
from basecampy3 import Basecamp3
bc3 = Basecamp3()
find_project = bc3.projects.find('Test Project')
find_project[0]
```

**Comment by phistrom** [comment]
Does it print a stack trace along with the error?

**Comment by phistrom** [comment]
Nevermind. Pretty sure I see exactly where the regression occurred. Sorry about that. I'll try to have a fix pushed up to PyPI tomorrow.

**Comment by phistrom** [comment]
Fixed in 0.6.1. Available on PyPI now. 

`pip install --upgrade basecampy3`

Thanks for the report!
